controllers/blobstorage/blobstorage_controller.go

```diff
@@ -72,7 +73,11 @@ func New(mgr manager.Manager) (*BlobStorageReconciler, error) {
 	if err != nil {
 		return nil, err
 	}
-	providerList := []providers.BlobStorageProvider{awsBlobStorageProvider, openshift.NewBlobStorageProvider(client, logger)}
+	providerList := []providers.BlobStorageProvider{
+		openshift.NewBlobStorageProvider(client, logger),
+		awsBlobStorageProvider,
+		gcp.NewGCPBlobStorageProvider(client),
```

Comment:
@cecobask , Maybe before using Openshift and GCP  BlobStorage Providers need check them, similar to AWS? (L72) --   awsBlobStorageProvider, err := aws.NewAWSBlobStorageProvider(client, logger))
 + error checking to be more informative - print provider name, in case of error

Reply:
@valerymo,

`NewAWSBlobStorageProvider()`, `NewAWSPostgresProvider()` and `NewAWSRedisProvider()` are the only provider methods that can return errors when being initialized because the underlying [credential manager](https://github.com/cecobask/cloud-resource-operator/blob/MGDAPI-4539/pkg/providers/aws/credentials.go#L145-L163) is likely to throw an error if there was a problem with the STS secret.
The `openshift` and `gcp` providers don't throw errors when they're being initialized, therefore there is nothing to check for. Maybe in the future this will change and when it does, error checking will definitely be introduced.